src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -20,4 +20,7 @@
     <ProjectReference Include="..\Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv\Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="@(CoreFXLabCompile)" />
```

Comment:
Does this do anything in this file? Where is the CoreFXLabCompile group defined?